src/BookNavigator/BookNavigator.js

```diff
@@ -1,17 +1,15 @@
+// eslint-disable-next-line no-unused-vars
+import { SharedResizeObserver } from '@internetarchive/shared-resize-observer';
```

Comment:
I don't think you need to import this since you're just getting a `SharedRO` passed-in from the consumer and not instantiating one yourself. In the TS world, you could use the `SharedROInterface`, but I don't think you need the import here.

Reply:
thanks, i am importing so i can set as type.